pyaedt/modules/AdvancedPostProcessing.py

```diff
@@ -497,9 +497,17 @@ def plot_field(
             ``"CutPlane"``, ``"Surface"``, and ``"Volume"``.
         setup : str, optional
             Setup and sweep name on which create the field plot. Default is None for nominal setup usage.
-        intrinsics : dict, optional.
-            Intrinsic dictionary that is needed for the export.
-            The default is ``None`` which try to retrieve intrinsics from setup.
+        intrinsics : dict, str, optional
+            This parameter represents the value of the intrinsics variable of the simulation
+            (typically Freq, Time, Phase) that are needed to compute the field before the export.
+            It can be a dictionary with keys depending of the design solution type.
```

Comment:
```suggestion
            It can be provided either as a dictionary or as a string.
            If it is a dictionary, keys depend on the solution type and can be expressed with both lower and camel case as:
            - ``"Freq"`` or ``"Frequency"``
            - ``"Time"``
            - ``"Phase"``
```